src/lib/viewers/doc/_docBase.scss

```diff
@@ -353,13 +353,10 @@ $thumbnail-sidebar-width: 226px;
                 display: block;
             }
 
-            .textLayer {
-                span {
-                    pointer-events: none;
-                }
-            }
-
-            .annotationLayer {
+            .annotationLayer,
+            .ba-HighlightList .ba-HighlightSvg.is-listening .ba-HighlightTarget-rect,
```

Comment:
I don't love how specific this rule has to be to override the existing styles, but this can be updated if we can make the underlying style less specific